public/pages/CreateSnapshotPolicy/containers/CreateSnapshotPolicy/CreateSnapshotPolicy.tsx

```diff
@@ -139,30 +144,28 @@ export class CreateSnapshotPolicy extends MDSEnabledComponent<CreateSMPolicyProp
       repoError: "",
       minCountError: "",
       timezoneError: "",
+      useNewUX: useNewUx,
```

Comment:
Perhaps move it to a context state?

Reply:
@SuZhou-Joe I'm of the opinion that its better to check for feature flag in `main.tsx` and pass it as a prop to components, we won't need context since its just a single level prop drilling. Wdyt?

Reply:
I am just thinking we are copy
```
const uiSettings = getUISettings();
const useNewUx = uiSettings.get("home:useNewHomePage");
```
in multiple place, maybe we can consider to move that to a util or somewhere.